06_create_cluster.sh

```diff
@@ -21,6 +24,19 @@ if [[ "${PULL_SECRET}" != *"cloud.openshift.com"* ]]; then
     exit 1
 fi
 
+if [ "$NODES_PLATFORM" = "assisted" ]; then
```

Comment:
I think perhaps it would be cleaner to refactor this so there's a new script e.g create_cluster_assisted.sh or similar, with any common parts moved to utils.sh as functions?